### PR TITLE
Implement FAT-17311

### PR DESCRIPTION
**Pull request by IhorBohdan**
***
[FAT-17311](https://issues.folio.org/browse/FAT-17311)
***
[C445994](https://foliotest.testrail.io/index.php?/cases/view/445994)
***
Screenshot 2024-11-12 110146

***

### CODE REVIEW

**Review by sviatlana-stsiapanava — APPROVED** [review]

File: cypress/e2e/finance/groups/decrease-allocation-for-group-financial-summary-is-displayed-correctly.cy.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    { tags: ['criticalPath', 'thunderjet'] },
+    () => {
+      FinanceHelp.searchByName(firstFund.name);
+      Funds.selectFund(firstFund.name);
+      Funds.selectBudgetDetails();
+      const amount = '10';
+
+      Funds.openMoveAllocationModal();
+      Funds.fillInAllAllocationFields(firstFund, secondFund, amount);
+      cy.pause();
```
Comment (sviatlana-stsiapanava) [inline cypress/e2e/finance/groups/decrease-allocation-for-group-financial-summary-is-displayed-correctly.cy.js:102]: forgot to remove cy.pause()

**Review by Sherzod-Kenjaev — APPROVED** [review]

File: cypress/e2e/finance/groups/decrease-allocation-for-group-financial-summary-is-displayed-correctly.cy.js (hunk 1 above)
Comment (ostapwd) [inline cypress/e2e/finance/groups/decrease-allocation-for-group-financial-summary-is-displayed-correctly.cy.js:102]: cy.pause();

**Review by ostapwd — CHANGES_REQUESTED** [review]
cy.pause();

File: cypress/e2e/finance/groups/decrease-allocation-for-group-financial-summary-is-displayed-correctly.cy.js (hunk 1 above)
Comment (IhorBohdan) [reply cypress/e2e/finance/groups/decrease-allocation-for-group-financial-summary-is-displayed-correctly.cy.js:102]: removed

Comment (IhorBohdan) [reply cypress/e2e/finance/groups/decrease-allocation-for-group-financial-summary-is-displayed-correctly.cy.js:102]: removed

**Pull request merged**